skbeam/core/correlation.py

```diff
@@ -336,18 +334,21 @@ def lazy_multi_tau(image_iterable, num_levels, num_bufs, labels,
 
         g2 = (s.G[:g_max] / (s.past_intensity[:g_max] *
                              s.future_intensity[:g_max]))
-        yield results(g2, lag_steps[:g_max], s)
+        yield results(g2, s.lag_steps[:g_max], s)
 
 
 def multi_tau_auto_corr(num_levels, num_bufs, labels, images):
     """Wraps generator implementation of multi-tau
 
-    See docstring for lazy_multi_tau
+    Original code(in Yorick) for multi tau auto correlation
+    @author: Mark Sutton
```

Comment:
I am not aware of `@author` meaning anything special.  Can you just have this say `Original code written by Mark Sutton`